### DISCUSSION TITLE
FontAwesome - PRO

### DISCUSSION THREAD

**Discussion by ekitm** [primefaces/community#253] [PrimeFaces] [answered]
How can i integrate Fontawesome - PRO.
I dont have a jar from fontawesome to include this in pom.

With following in pom, i include the free-version:
```
    <dependency>
      <groupId>org.webjars</groupId>
      <artifactId>font-awesome</artifactId>
      <version>6.3.0</version>
    </dependency>
```

**Comment by tandraschko** [comment, marked as answer]
i dont know about font-awesome PRO but i think your downloaded a zip archive?
just unzip it in your src/main/webap and include the CSS in your template - just plain HTML style, not JSF/PF special stuff here
